### PR TITLE
Update instructions for labeling Jira tickets + Update translations PR reviewers

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cwarnermm — DISMISSED** [review]
Great update, @amyblais! Thank you!

File: operations/research-and-development/product/development-process/new-bug-tickets/README.md [hunk 1]
```diff
@@ -18,8 +18,10 @@ Bugs are any “obvious errors” on how the product or a feature is functioning
 
 ## 4. Information needed on Mattermost bug tickets
 
-* **Labels:** Add a `customer-bug` label if the ticket is based on a customer bug report. Add a `community-bug` label if the ticket is based on a community bug report.
-* **Environment:** There is an ``Environment`` field with drop-down options for whether the bug was found in Cloud test servers, Cloud production servers, Self-Hosted test servers, Self-Hosted production servers, or in Master/PR testing. Additionally, in the ticket description you can add details on Mattermost server and version, OS and version, Mattermost mobile app version, Mattermost desktop app version, any notable Mattermost configurations \(such as HA, Elasticsearch, image proxy, SSO\).
+* **Labels:** Add a label based on the following criteria:
+  - `prerelease-bug` label if the ticket is based on a bug found on master/main or during day-to-day use on community.mattermost.com.
+  - `rc-bug` label if the ticket is based on a bug found during release testing.
+   - `customer-bug` or `community-bug` label if the ticket is based on a customer or a community bug report on production.
```
Comment (saturninoabril) [inline operations/research-and-development/product/development-process/new-bug-tickets/README.md]:
```suggestion
  - `rc-bug` label if the ticket is based on a bug found during release testing.
  - `customer-bug` or `community-bug` label if the ticket is based on a customer or a community bug report on production.
```

**Review by saturninoabril — DISMISSED** [review]
Thank you Amy!

(left small formatting request)

File: operations/research-and-development/product/development-process/new-bug-tickets/README.md [hunk 2]
```diff
@@ -18,8 +18,10 @@ Bugs are any “obvious errors” on how the product or a feature is functioning
 
 ## 4. Information needed on Mattermost bug tickets
 
-* **Labels:** Add a `customer-bug` label if the ticket is based on a customer bug report. Add a `community-bug` label if the ticket is based on a community bug report.
-* **Environment:** There is an ``Environment`` field with drop-down options for whether the bug was found in Cloud test servers, Cloud production servers, Self-Hosted test servers, Self-Hosted production servers, or in Master/PR testing. Additionally, in the ticket description you can add details on Mattermost server and version, OS and version, Mattermost mobile app version, Mattermost desktop app version, any notable Mattermost configurations \(such as HA, Elasticsearch, image proxy, SSO\).
+* **Labels:** Add a label based on the following criteria:
+  - `prerelease-bug` label if the ticket is based on a bug found on master/main or during day-to-day use on community.mattermost.com.
+  - `rc-bug` label if the ticket is based on a bug found during release testing.
+  - `customer-bug` or `community-bug` label if the ticket is based on a customer or a community bug report on production.
```
Comment (saturninoabril) [inline operations/research-and-development/product/development-process/new-bug-tickets/README.md]: In addition, I wonder if we could include the points as listed by Linda at https://community.mattermost.com/private-core/pl/7q9c3qd9cpdk3rwix5a8yiwjjw. I feel like it's a good guide for everyone to classify a bug.

**Review by lindalumitchell — DISMISSED** [review]
Thanks Amy, LGTM!

**Review by saturninoabril — DISMISSED** [review]
Thanks Amy, LGTM!

**Comment by amyblais** [comment]
Applied edits based on the latest discussion https://community.mattermost.com/private-core/pl/uo8ca19w7781xj1cjjcotg1xua.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**